modules/trusted-firmware-m/Kconfig

```diff
@@ -20,6 +20,13 @@ config TFM_BL2
 	bool
 	default n
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
+	default y
```

Comment:
I believe we should restrict this based on the SoC, nRF9160 and nRF5340, as those are the only platforms defaulting SECURE_UART1 to `ON` in TF-M.

Reply:
Isn't that an sdk-nrf specific change now? So why do we need to consider other boards in this case?

Reply:
because you can still build for other boards, just not use the nRF specific features.

Reply:
That should be fine now I think.